library/src/main/java/com/novoda/downloadmanager/DownloadsBatchPersistence.java

```diff
@@ -19,17 +19,19 @@
     private final DownloadsPersistence downloadsPersistence;
     private final CallbackThrottleCreator callbackThrottleCreator;
     private final ConnectionChecker connectionChecker;
+    private final DownloadBatchRequirementRule downloadBatchRequirementRule;
 
     DownloadsBatchPersistence(Executor executor,
                               DownloadsFilePersistence downloadsFilePersistence,
                               DownloadsPersistence downloadsPersistence,
                               CallbackThrottleCreator callbackThrottleCreator,
-                              ConnectionChecker connectionChecker) {
+                              ConnectionChecker connectionChecker, DownloadBatchRequirementRule downloadBatchRequirementRule) {
```

Comment:
One parameter per line 😄

Reply:
Ok, I will correct all formatting stuff :) . Btw. I saw your formatting rules are defined https://github.com/novoda/novoda/tree/master/ide-settings. I will apply it to my IDE